packages/ui/src/lib/Button.svelte

```diff
@@ -11,6 +11,7 @@
 		// Layout props
 		shrinkable?: boolean;
 		reversedDirection?: boolean;
+		height?: number | string | undefined;
```

Comment:
How come you want to allow string, unlike width which is either undefined or a number?

Reply:
For "auto" or "100%" and the like, but maybe not necessary. I'll double check this and either pull it or make width also take it so it's consistent